src/modules/ctc/src/ctc_callback_handler.cc

```diff
@@ -0,0 +1,181 @@
+#include "ctc_callback_handler.h"
+
+#include <string>
+
+#include "channel.h"
+#include "launcher.h"
+
+namespace ctc
+{
+
+// TODO repeat for each event listener
```

Comment:
Add a ticket for any TODOs and put the ticket number in each TODO comment.

Reply:
Ticket Added